CHANGELOG.md

```diff
@@ -2,6 +2,86 @@
 
 This document outlines major changes between releases.
 
+## 0.99.5 "Underestimation" (11 Nov 2022)
+
+A long time has passed since the previous release, a juicy set of features and
+optimisations came on board, so we've decided to change our release plan and
+supply one more 3.4.0 compatible release. It comes up with a massive suite
+of network P2P and broadcast improvements. We've carefully investigated message
+sending timeouts on benchmarks, reworked the broadcast logic and significantly
+reduced delays in message sending and amount of useless traffic between our
+nodes. For smart-contract developers and RPC client users several new features
+were added. These are automatic RPC bindings generator, poll-based and
+websocket-based transactions awaiting and extended web-socket client
+subscriptions interface. For better debugging experience we've improved state
+related functionality of VM CLI and added an ability to reset chain state to a
+particular height.
+
+An important dBFT initialization bug that caused timestamp-related errors on
+block addition was fixed. Also, those who use NeoGo node with
+EnableCORSWorkaround setting may want to update to be able to handle web-socket
+requests with Origin set in header. Finally, we're kindly asking our users to
+make acquaintance with our roadmap file where deprecated code removal schedule
+is placed.
+
+This version is fully compatible with C# node 3.4.0 and does not require
+resynchronization on upgrade. This time for sure, the next release is planned
+to be compatible with 3.5.0.
+
+New features:
+ * logarithmic gossip fan-out, new `BroadcastFactor` application setting and
+   adaptive peer requests are added to optimise broadcasting process (#608,
```

Comment:
Usually I'm using PR numbers only, but issues are OK too.

Reply:
I didn't include all issues, but some of them contain useful information or duscussions, thus I've included thim. Although I'm not sure about it, maybe we really need PRs only to be present here.

Reply:
It's not a strict policy, both can be referenced if needed.